otx/algorithms/detection/adapters/mmdet/datasets/task_adapt_dataset.py

```diff
@@ -9,6 +10,7 @@
 from otx.mpa.modules.utils.task_adapt import map_cat_and_cls_as_order, map_class_names
 
 
+# pylint: disable=invalid-name
```

Comment:
What is the invalid name? In my experience, this kind of pylint issue can be solved easily

Reply:
This is an issue with self.CLASSES in the Dataset Class, and since it's also related to other Datasets, i've left it as is for now.